c3pzero_description/CMakeLists.txt

```diff
@@ -0,0 +1,29 @@
+cmake_minimum_required(VERSION 3.5)
+project(c3pzero_description)
+
+if(CMAKE_COMPILER_IS_GNUCXX OR CMAKE_CXX_COMPILER_ID MATCHES "Clang")
+  add_compile_options(-Wall -Wextra -Wpedantic)
+endif()
+
```

Comment:
```suggestion
```

Reply:
I would like to keep warnings here. I fixed the rest.

Reply:
Keep the warnings even though no code is being compiled? I would have recommended this change if the warnings were actually being used.

Reply:
Wrong package my bad, this has been removed